src/style.css

```
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+.searchContainer {
+  margin-top: 20px;
+}
+
+.searchForm {
+  --text-color: #3d4e61;
+  --font: avenir, helvetica, sans-serif;
+  --gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search {
+  width: 100%;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-weight: 900;
+}
+
+.bigSearch {
+  padding: 23px 20px 23px 62px;
+  line-height: 22px;
+  font-size: 16px;
+  background-position: 25px 24px;
+  background-size: 19px;
+}
+
+.smallSearch {
+  padding: 11px 10px 10px 33px;
+  line-height: 19px;
+  font-size: 14px;
+  background-position: 12px 14px;
+  background-size: 11px;
+}
+
+.search::placeholder {
+  font-family: var(--font);
```

Redundant property, placeholder inherits font-family from it's parent
```suggestion
  font-family: var(--font);
```